include/boost/url/impl/url.ipp

```diff
@@ -870,6 +905,13 @@ set_port(string_view s)
     return *this;
 }
 
+url&
+url::
+set_port(pct_encoded_view s)
```

Comment:
You did more work than necessary.. port can't be encoded:
```
port        = *DIGIT
```

Did we even have `set_port( string_view )` before?

Reply:
I removed `set_port(pct_encoded_view s);`

> Did we even have set_port( string_view ) before?

Yes. `set_port( string_view )` should be useful. Ports in Asio applications are often strings.

Reply:
Fair, but note that the string passed to `set_port( string_view )` must only contain digits (or be empty).